pages/advocates.vue

```diff
@@ -20,14 +20,14 @@ definePageMeta({
   routeName: "advocates",
 });
 
-useHead({
+useSeoMeta({
   title: "Qiskit Advocates",
-  meta: [
-    {
-      name: "description",
-      content:
-        "The Qiskit advocate program is a global program that provides support to the individuals who actively contribute to the Qiskit Community.",
-    },
-  ],
+  ogTitle: "Qiskit Advocates",
+  description:
+    "The Qiskit advocate program is a global program that provides support to the individuals who actively contribute to the Qiskit Community. Join the program today!",
+  ogDescription:
+    "The Qiskit advocate program is a global program that provides support to the individuals who actively contribute to the Qiskit Community. Join the program today!",
+  ogImage: "/images/qiskit-logo.png",
```

Comment:
one thing I forgot: if we want this image to show up in tweet cards, we'll need to use the full URL

`https://qiskit.org/images/qiskit-logo.png`

Using your [preview URL](https://qiskit-org-pr-3300.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events), and a tool like this [Twitter Card Validator](https://tweetpik.com/twitter-card-validator), you'll see that there is no image preview in the twitter card.

![Screenshot 2023-06-12 at 6 11 47 AM (2)](https://github.com/Qiskit/qiskit.org/assets/6276074/504e17a4-47c3-43b2-957d-0f7db9a55e34)


However, if you use the full img URL like what was done in the Summer School 2023 page [here](https://github.com/Qiskit/qiskit.org/blob/social-media-tags/pages/events/summer-school-2023.vue#L123-L124), you can see the tweet card produces an image... 

![Screenshot 2023-06-12 at 6 12 01 AM (2)](https://github.com/Qiskit/qiskit.org/assets/6276074/c704b361-3365-4442-8eb8-dea5ba011b36)